force-app/main/default/classes/customObjectController.cls

```diff
@@ -0,0 +1,54 @@
+public with sharing class customObjectController {
+    @AuraEnabled(cacheable=true)
+    public static List<Graduate_Student__c> getGraduateStudentList(){
+        return [SELECT Id, Name, Degree__c FROM Graduate_Student__c];
+    }
+
+
+    @AuraEnabled
+    public static void deleteRecord(String recordId) {
+       
+        Graduate_Student__c student = [SELECT Id, Name, Degree__c FROM Graduate_Student__c WHERE Id = :recordId];
+        if(student != null) {
```

Comment:
only success check is here. need to add the unsuccessful operation check as well